pytest_subtests.py

```diff
@@ -15,6 +15,17 @@
 from _pytest.unittest import TestCaseFunction
 
 
+def pytest_addoption(parser):
+    group = parser.getgroup("subtests")
+    group.addoption(
+        "--disable-shortletter-output",
```

Comment:
I think we should make this more obvious that is related to subtests, how about naming it `--no-subtests-shortletter`?

Reply:
Sounds reasonable to me!